src/components/nowPlayingBar/nowPlayingBar.scss

```diff
@@ -58,6 +58,7 @@
     font-size: 92%;
     margin-right: 1em;
     margin-left: 0.5em;
+    cursor: pointer;
```

Comment:
Taking into account [this](https://github.com/jellyfin/jellyfin-web/blob/396f4b1ed2168986785f0d233c26b043d87c8ed3/src/components/nowPlayingBar/nowPlayingBar.js#L244-L248), we should move `cursor` CSS to `.nowPlayingBar`.

Reply:
You are right. Sorry, I didnt dive very deep into the source. I changed it now.